src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -245,14 +245,14 @@ void parseCvrFile(String filePath, List<CastVoteRecord> castVoteRecords) {
   }
 
   // Simple container class for contest data
-  private static class Contest {
+  public static class Contest {
 
-    private final String name;
-    private final Integer id;
-    private final Integer numCandidates;
-    private final Integer maxRanks;
+    public String name;
+    public Integer id;
+    public Integer numCandidates;
+    public Integer maxRanks;
 
-    private Contest(String name, Integer id, Integer numCandidates, Integer maxRanks) {
+    public Contest(String name, Integer id, Integer numCandidates, Integer maxRanks) {
```

Comment:
I don't think anything in this entire project should be `public` unless it's being directly accessed by the Jackson library (e.g. `RawContestConfig`).

The first place I see this being accessed is in `writeGenericCvrCsv`. It only looks like `maxRanks` and `id` are retrieved, so I'd write project-visible getters for these.

The second place I see this being accessed is in the `TabulatorSession` line: `dominionCvrReader.contests.values()`.

I've never seen `.values()` of a class used like that before, but for safety's sake, I'd lean strongly towards keeping all these `private` and `private final` and maybe making a project-visible method that explicitly accesses, formats, and returns these values in the way that makes `writer.writeGenericCvrCsv` happy (i.e. a `Collection<Contest>`).